tekton/cronjobs/dogfooding/releases/cel-nightly/README.md

```diff
@@ -0,0 +1,2 @@
+Cron Job to trigger the Tekton Pipeline cel nightly build.
+Results are published to https://storage.cloud.google.com/tekton-releases-nightly/task-loops/latest/release.yaml
```

Comment:
did you mean to update this form `task-loops` to `cel`?

Reply:
fixed